h2o-core/src/main/java/water/jdbc/SQLManager.java

```diff
@@ -16,6 +16,7 @@
 
   private static final String TEMP_TABLE_NAME = "table_for_h2o_import";
   private static final String MAX_USR_CONNECTIONS_KEY = H2O.OptArgs.SYSTEM_PROP_PREFIX + "sql.connections.max";
```

Comment:
Inconsistent with my prefix. I'd adjust my variable to sql.jdbc.connections.max maybe ? If we haven't released it yet !

Reply:
Why do you think it is inconsistent?

IMHO the prefix is "sql" for SQLManager

"sql.connections" - is for number of connections in SQLManager, not necessarily a property of the JDBC interface
"sql.jdbc" - for properties of the jdbc driver

I would not change yours. We can change mine, please suggest! :)

Reply:
In that case, your naming is just fine ! Completely valid.